src/components/Header.svelte

```diff
@@ -2,13 +2,11 @@
   import { goto } from '$app/navigation'
   import { sessionStore, themeStore } from '../stores'
   import { DEFAULT_THEME_KEY, storeTheme, type ThemeOptions } from '$lib/theme'
-  import AlphaTag from '$components/nav/AlphaTag.svelte'
   import Avatar from '$components/settings/Avatar.svelte'
-  import BrandLogo from '$components/icons/BrandLogo.svelte'
-  import BrandWordmark from '$components/icons/BrandWordmark.svelte'
   import DarkMode from '$components/icons/DarkMode.svelte'
   import Hamburger from '$components/icons/Hamburger.svelte'
   import LightMode from '$components/icons/LightMode.svelte'
+  import Logo from './icons/Logo.svelte'
```

Comment:
maybe import it from `$components/icons/Logo.svelte` so it's consistent with the other imports? not a big deal tho

Reply:
Oh yeah, good call! 💯 Overlooked that when bringing in the new logo.